bundles/org.openhab.binding.ferroamp/README.md

```diff
@@ -0,0 +1,239 @@
+| s2-dc-link-voltage                    | Number:ElectricPotential   | R          | S2 DC Link Voltage                       | DC link voltage as measured by SSO-2 |
+| s2-timestamp                          | DateTime                   | R          | S2 Time Stamp When Message was Published | Time stamp when message was published |
+ |
+
+| s3-id                                 | String                     | R          | S3 ID                                    | Unique identifier of SSO-3 |
+| s3-measured-voltage-on-pv-string-side | Number:ElectricPotential   | R          | S3 Measured Voltage on PV String Side    | Measured on PV string side |
+| s3-measured-current-on-pv-string-side | Number:ElectricCurrent     | R          | S3 Measured Current on PV String Side    | Measured on PV string side |
+| s3-total-solar-energy                 | Number:Energy              | R          | S3 Total Solar Energy                    | Total energy produced by SSO-3 |
+| s3-relay-status                       | String                     | R          | S3 Relay Status                          | 0 = relay closed (i.e running power), 1 = relay open/disconnected, 2 = precharge |
+| s3-temperature                        | Number:Temperature         | R          | S3 Temperature Measured on PCB           | Temperature Measured on PCB |
+| s3-fault-code                         | String                     | R          | S3 FaultCode                             | 0x00 = OK. For all other values Please contact Ferroamp support |
+| s3-dc-link-voltage                    | Number:ElectricPotential   | R          | S3 DC Link Voltage                       | DC link voltage as measured by SSO-3 |
+| s3-timestamp                          | DateTime                   | R          | S3 Time Stamp When Message was Published | Time stamp when message was published |
+ |
+
+| eso-unique-identifier         | String                     | R          | Eso Unique Identifier                    | Unique identifier |
+| measured-voltage-battery      | Number:ElectricPotential   | R          | Eso Voltage Measured on Battery Side     | Measured on battery side |
+| measured-current-battery      | Number:ElectricCurrent     | R          | Eso Current Measured on Battery Side     | Measured on battery side |
+| eso-battery-energy-produced   | Number:Energy              | R          | Eso Battery Energy Produced              | Total energy produced by ESO, i.e total energy charged |
+| eso-battery-energy-consumed   | Number:Energy              | R          | Eso Battery Energy Consumed              | Total energy consumed by ESO, i.e total energy discharged |
+| eso-soc                       | Number:Dimensionless       | R          | Eso State of Charge                      | State of Charge for ESO |
+| eso-relay-status              | String                     | R          | Eso Relay Status                         | 0 = relay closed, 1 = relay open |
```

Comment:
Maybe change this to a Contact itemtype.

Reply:
Done